tests/unit/test_client.py

```diff
@@ -1252,38 +1394,64 @@ class _Entity(dict):
 
 
 class _Key(object):
-    _MARKER = object()
```

Comment:
`reserve_ids_sequential` required a mock `Key` class that was closer to the real thing.